resources/app/log-collector.yaml

```diff
@@ -0,0 +1,148 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: log-collector
+  namespace: kube-system
+  labels:
+    app: log-collector
+spec:
+  type: ClusterIP
+  ports:
+    - name: rest
```

Comment:
nit: I swear I spent a good minute trying to figure out what this meant. I'd suggest `api` or `web`

Reply:
renamed to api, applied (not yet merged to this branch)